<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1150</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>115</horstretch>
    <verstretch>80</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="GLVIew" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>781</height>
     </rect>
    </property>
   </widget>
   <widget class="QSlider" name="scaleSlider">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>200</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="sliderPosition">
     <number>100</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="scaleLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>200</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QLabel" name="positionSectionLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>70</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Position</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xPositionEdit">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="xPositionLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>100</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>X:</string>
    </property>
   </widget>
   <widget class="QLabel" name="yPositionLabel">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>100</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Y:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yPositionEdit">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="zPositionLabel">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>100</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Z:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zPositionEdit">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="yRotationEdit">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="zRotationLabel">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>160</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OZ:</string>
    </property>
   </widget>
   <widget class="QLabel" name="xRotationLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>160</y>
      <width>20</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OX:</string>
    </property>
   </widget>
   <widget class="QLabel" name="rotationSectionLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>130</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotation around:</string>
    </property>
   </widget>
   <widget class="QLabel" name="yRotationLabel">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>160</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OY:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xRotationEdit">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="zRotationEdit">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>160</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="filenameLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>10</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>No file opened</string>
    </property>
   </widget>
   <widget class="QLabel" name="verticesCountLabel">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="poligonsCountLabel">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>30</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>230</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="projectionParallelButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>250</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Parallel</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="projectionCentralButton">
    <property name="geometry">
     <rect>
      <x>1019</x>
      <y>250</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Central</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="projectionSelectionLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>260</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Projection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="solidLineButton">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>310</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Solid</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="dottedLineButton">
    <property name="geometry">
     <rect>
      <x>1019</x>
      <y>310</y>
      <width>111</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Dotted</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lineTypeSelectionLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>320</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Lines</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edgesColorREdit">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>390</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="edgesColorRLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>390</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R:</string>
    </property>
   </widget>
   <widget class="QLabel" name="edgesColorGLabel">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>390</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>G:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edgesColorGEdit">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>390</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="edgesColorBEdit">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>390</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="edgesColorBLabel">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>390</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>B:</string>
    </property>
   </widget>
   <widget class="QLabel" name="vertexSelectionLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>450</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Vertices</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vertexTypeSquareButton">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>440</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Square</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="vertexTypeCircleButton">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>440</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Circle</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="vertexTypeNoneButton">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>440</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>290</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="lineSizeSlider">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>350</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>3</number>
    </property>
    <property name="sliderPosition">
     <number>3</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="sizeSliderLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>350</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>420</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticesSizeSlider">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>480</y>
      <width>261</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
    <property name="sliderPosition">
     <number>10</number>
    </property>
    <property name="tracking">
     <bool>true</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="verticesSizeSliderLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>480</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Size</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="verticesColorBEdit">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>520</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="verticesColorGLabel">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>520</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>G:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="verticesColorREdit">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>520</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="verticesColorGEdit">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>520</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="verticesColorBLabel">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>520</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>B:</string>
    </property>
   </widget>
   <widget class="QLabel" name="verticesColorRLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>520</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R:</string>
    </property>
   </widget>
   <widget class="QLabel" name="verticesCountLabelLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>30</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vertices:</string>
    </property>
   </widget>
   <widget class="QLabel" name="poligonsCountLabelLabel">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>30</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Poligons:</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>550</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="backGroundSectionLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>570</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Background</string>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundColorGLabel">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>600</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>G:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="backGroundColorGEdit">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>600</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="backGroundColorREdit">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>600</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundColorBLabel">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>600</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>B:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="backGroundColorBEdit">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>600</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="inputMask">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="backgroundColorRLabel">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>600</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>R:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="renderButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>730</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Render Scene</string>
    </property>
   </widget>
   <widget class="QPushButton" name="snapButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>700</y>
      <width>311</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Take a snap</string>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>680</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1150</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionChoose_File"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionSet_to_center"/>
    <addaction name="actionObject_color"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionChoose_File">
   <property name="text">
    <string>Choose File</string>
   </property>
  </action>
  <action name="actionSet_to_center">
   <property name="text">
    <string>Set to center</string>
   </property>
  </action>
  <action name="actionObject_color">
   <property name="text">
    <string>Object color</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLVIew</class>
   <extends>QOpenGLWidget</extends>
   <header location="global">glview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
